src/modules/accelerometer.cpp

```diff
@@ -117,6 +117,10 @@ namespace Modules
 
             newFrame.time = DriversNRF::Timers::millis();
             newFrame.jerk = ((newFrame.acc - lastFrame.acc) * 1000.0f) / (float)(newFrame.time - lastFrame.time);
+            //if the time between last and current time is close to zero, log it
+            if(newFrame.time - lastFrame.time < 1){
+                NRF_LOG_INFO("time difference between newFrame and lastFrame is 0, newFrame.time: %d, lastFrame.time: %d", newFrame.time, lastFrame.time);
```

Comment:
This comparison is same as `newFrame.time - lastFrame.time == 0` (since the numbers are unsigned integers), if that was your intention then it's best to use == 0 rather than < 1 for clarity